builtin/credential/github/path_config.go

```diff
@@ -94,7 +94,13 @@ func (b *backend) pathConfigWrite(ctx context.Context, req *logical.Request, dat
 	}
 
 	if c.OrganizationID == 0 {
-		client, err := b.Client("")
+		var githubToken string
+		if token, ok := data.GetOk("token"); ok {
+			if t, ok := token.(string); ok {
```

Comment:
We should add `token` to the `framework.FieldSchema` above in `pathConfig()`. You can see an example of that [here](https://github.com/hashicorp/vault/blob/0c2fadca9e7aafa23fd8f26cbac992fd16b8cd63/builtin/credential/github/path_login.go#L20-L23). If we do that then we can safely do the type assertion without this "if ok" pattern. Additionally, we will want to add `token` to the API docs.